.github/workflows/tests_windows.yml

```
@@ -75,7 +75,7 @@ jobs:
       - name: Install dependencies
         if: steps.kokkos-cache.outputs.cache-hit != 'true'
         run: |
-          python -m pip install cmake build
+          python -m pip install "cmake!=3.29.0" build
```

The default Python version in this action is > 3.9. We may don't need this change here,
```suggestion
          python -m pip install cmake build
```